### ISSUE TITLE
Circle display disappears when tasks are unchecked

### ISSUE THREAD

**Issue by hmansoori** [breakbreakdown/webapp#17]
The circle display in the middle goes away whenever there is an unchecked task, but there is no effect on the overall free time in the day. This happened as soon as I created a new task or event and it appeared in the checklist after I refreshed the page. The circle reappears only after each item in the checklist has been checked. I tried this with a mix of tasks and events and still received the same results. This occurred in a regular chrome session, and I had already approved all google pop-ups and successfully logged in.
![image](https://user-images.githubusercontent.com/22490092/40929441-8917ea80-67d9-11e8-8355-f629ba7ee324.png)

![image](https://user-images.githubusercontent.com/22490092/40929464-9bcfad5c-67d9-11e8-8f70-f5aa21572b5b.png)

**Comment by nogawb** [comment]
This bug should be fixed as we fixed the underlying data issues, however we did have trouble replicating the front end bug. If it is still happening for you please let us know. We appreciate your detailed feedback and instructions for replication.

**Comment by abegel** [comment]
@hmansoori Can you replicate this bug, Hannah?

**Comment by hmansoori** [comment]
I just tried to. I logged in the same way as before and created a similar style task. It looks like the issue has been fixed though! The circle display is there and is properly displaying my tasks based on whether they're checked off or not.
